src/libs/helpers/state-manipulator/spec-manipulator.ts

```diff
@@ -11,6 +11,7 @@ export interface SpecOptions {
   relayChain?: string;
   chainType?: string;
   paraId?: number;
+  soloChain?: boolean;
```

Comment:
maybe using devService or devChain maybe instead of solo

Reply:
Done